### PR TITLE
Allow users to send email report upon crash

### PR DESCRIPTION
**Pull request by dippman**
During runtime crashes, user has the option to send crash info to mm_support@chem.wisc.edu from their local email client.

### CODE REVIEW

File: GUI/MainWindow.xaml.cs [hunk 1]
```diff
@@ -427,8 +428,26 @@ private void EverythingRunnerExceptionHandler(Task obj)
...
-                MessageBox.Show(message);
+                var messageBoxResult = System.Windows.MessageBox.Show(message + "\n\nWould you like to report this crash?", "Runtime Error", MessageBoxButton.YesNo);
                 outRichTextBox.AppendText(message + Environment.NewLine);
+                Exception exception = e;
+
+
+                if(messageBoxResult == MessageBoxResult.Yes)
+                {
+                    string body = exception.Message + "%0D%0A" + exception.Data +
+                       "%0D%0A" + exception.StackTrace +
```
Comment (stefanks) [inline GUI/MainWindow.xaml.cs]: What's %0D%0A

File: TaskLayer/SearchTask/SearchTask.cs [hunk 2]
```diff
@@ -1094,6 +1092,7 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
 
             var psmsGroupedByFile = allPsms.GroupBy(p => p.FullFilePath);
 
+            Psm.ModstoWritePruned = SearchParameters.ModsToWriteSelection;
```
Comment (stefanks) [inline TaskLayer/SearchTask/SearchTask.cs]: No, revert this change please!

File: TaskLayer/TaskLayer.csproj [hunk 3]
```diff
@@ -18,6 +18,10 @@
     <ProjectReference Include="..\EngineLayer\EngineLayer.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Windows.Forms" />
+  </ItemGroup>
+
```
Comment (stefanks) [inline TaskLayer/TaskLayer.csproj]: Is this needed in the task layer?

File: GUI/MainWindow.xaml.cs (hunk 1 above)
Comment (dippman) [reply GUI/MainWindow.xaml.cs]: These are the characters for 'newline' needed to format with the line endings that emails use. For some reason '\n' , '\r\n', and 'Environment.newline'  don't work for email line endings.

File: TaskLayer/TaskLayer.csproj (hunk 3 above)
Comment (dippman) [reply TaskLayer/TaskLayer.csproj]: woops, no
